java/blockchain/src/test/java/org/learn/blockchains/BlockchainApplicationTests.java

```diff
@@ -0,0 +1,60 @@
+package org.learn.blockchains;
+
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.MalformedURLException;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.learn.blockchains.components.BlockChain;
+import org.learn.blockchains.model.Block;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
```

Comment:
current test cases don't need server to be started, they are just doing component level testing.

Reply:
I removed it. Thanks for the suggestion.